webpack.config.js

```diff
@@ -132,6 +125,16 @@ if ( CALYPSO_ENV === 'development' ) {
 
 	// Add react hot loader before babel-loader
 	jsLoader.loaders = [ 'react-hot' ].concat( jsLoader.loaders );
+
+	if ( config.isEnabled( 'use-source-maps' ) ) {
+		webpackConfig.debug = true;
+		webpackConfig.devtool = '#eval-cheap-module-source-map';
```

Comment:
might as well go with `eval-source-map` if we're going to make it optional. we'll get column level sourcemaps, which should fix some of the issues @aduth saw with hovering over variable names in the debugger.